mlflow/models/cli.py

```diff
@@ -188,6 +190,16 @@ def build_docker(model_uri, name, env_manager, mlflow_home, install_mlflow, enab
 
         docker run -p 5001:8080 -e DISABLE_NGINX=true "my-image-name"
 
+    If ``--model-uri`` is NOT specified:
+
+    Builds a Docker image that can serve an arbitrary MLflow model. When launching a container with
+    the built image, the model artifacts directory must be mounted as a volume into
+    the ``/opt/ml/model`` directory in the container as shown in the example below.
+
+    .. code:: bash
+
+        mlflow models build-docker -n my-image-name
+        docker run --rm -p 5001:8080 -v /path/to/artifacts/model:/opt/ml/model my-image-name
```

Comment:
Thoughts:

If ``--model-uri`` is NOT specified when buiding image,
when launching a container, we can read model-uri from a provided environment, instead of specifying a model mount directory like `-v /path/to/artifacts/model:/opt/ml/model`, WDYT ?

Reply:
e.g.

```
docker run --env MODEL_UIR=XXX ...
```

Reply:
oh, I understood, if model_uri is local file, then docker container cannot visit it unless we mount it.

Reply:
The image doesn't contain model artifacts.